### PR TITLE
Fixes #20269 - Use the  inline messaging warning

### PR DESCRIPTION
**Pull request by rabajaj0509**
This commit makes sure that the warning message follows the patternfly standards.

### CODE REVIEW

File: app/views/discovered_hosts/multiple_destroy.html.erb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <%= form_tag submit_multiple_destroy_discovered_hosts_path(:host_ids => params[:host_ids]), :onsubmit => "resetSelection()" do -%>
-  <span class="label label-warning"><%= _('Warning') -%></span>
-  <%= _('This might take a while, as all hosts, facts and reports will be destroyed as well') %>
-<% end %><br />
+  <div class="alert alert-warning">
```
Comment (ohadlevy) [inline app/views/discovered_hosts/multiple_destroy.html.erb]: please use the alert helper instead (https://github.com/theforeman/foreman/blob/develop/app/helpers/layout_helper.rb#L96)

**Comment by orrabin** [comment]
looks good:
image

**Comment by rabajaj0509** [comment]
@orrabin thanks :)

**Comment by lzap** [comment]
[test]

**Comment by rabajaj0509** [comment]
@lzap can you please re-run the tests, irrelevant failure in jenkins :(

**Comment by lzap** [comment]
[test]

**Comment by lzap** [comment]
Thanks a lot @rahulbajaj0509 ! Merged

**Pull request merged**